shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -11,6 +11,10 @@
 static const char _kTextAffinityDownstream[] = "TextAffinity.downstream";
 static const char _kTextAffinityUpstream[] = "TextAffinity.upstream";
 
+// The canonical invalid CGRect that if returned in firstRectForRange, the IME candidates view
+// will be hidden. Also used to indicate the first rect cache is now invalid.
+const CGRect kInvalidFirstRect = {{-1, -1}, {9999, 9999}};
```

Comment:
Does it result in nothing being shown onscreen when this is used, or is this what we had before with it being displayed in the top left corner?

Reply:
The former. It doesn't seem to add the candidates view to the view hierarchy when that `CGRect` is returned by `firstRectForRange`